pkg/alertmanager/alertstore/local/store.go

```diff
@@ -20,6 +20,7 @@ const (
 
 var (
 	errReadOnly = errors.New("local alertmanager config storage is read-only")
+	errState    = errors.New("legacy local alertmanager storage does not support state")
```

Comment:
The local read-only store is not legacy.

```suggestion
	errState    = errors.New("local alertmanager storage does not support state persistency")
```